sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteRecordsInconsistent.java

```diff
@@ -50,14 +49,16 @@ public PCollection<Void> expand(PCollection<KV<DestinationT, StorageApiWritePayl
     BigQueryOptions bigQueryOptions = input.getPipeline().getOptions().as(BigQueryOptions.class);
     // Append records to the Storage API streams.
     input.apply(
-        "Write Records",
```

Comment:
Changing transform names can affect update compatibility - do you need this?

Reply:
No I don't, but this does not follow the same convention other apply labels use (no spaces on names). Should I revert?